docs/src/content/docs/reference/configuration.mdx

```diff
@@ -447,16 +447,32 @@ When using custom themes and setting this to `true`, you must provide at least o
 
 ### `pagefind`
 
-**type:** `boolean`  
+**type:** <code>boolean | <a href="#pagefindoptions">PagefindOptions</a></code>  
 **default:** `true`
 
-Define whether Starlight’s default site search provider [Pagefind](https://pagefind.app/) is enabled.
+Configures Starlight’s default site search provider [Pagefind](https://pagefind.app/).
```

Comment:
Now I re-read this, actually this should probably be just “Configure” to match the other verbs.
```suggestion
Configure Starlight’s default site search provider [Pagefind](https://pagefind.app/).
```